examples/profile/profile_spec.rb

```diff
@@ -15,40 +15,54 @@
 
 describe 'sandbox example' do
   it 'returns a user profile' do
+    expiry_date = Time.new + 86_400
```

Comment:
Easy to guess, but might be clearer if we name this variable before using it, so it's clearer what it is:
```suggestion
    seconds_in_day = 86_400
    expiry_date = Time.new + seconds_in_day 
```